mindsdb/integrations/handlers/duckduckgozeroclick_handler/duckduckgozeroclick_handler.py

```diff
@@ -0,0 +1,98 @@
+import requests
+from typing import Dict, Any
+from collections import OrderedDict
+import pandas as pd
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.integrations.libs.response import RESPONSE_TYPE
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+
+class DuckDuckGoHandler(APIHandler):
+    def __init__(self, name: str = None, **kwargs):
+        super().__init__(name)
+        self.api_key = None
+        self.is_connected = False
+
+        args = kwargs.get('connection_data', {})
+        if 'api_key' in args:
+            self.api_key = args['api_key']
+
+    def connect(self) -> bool:
+        if self.api_key:
+            self.is_connected = True
+            return True
+        else:
+            return False
+
+    def check_connection(self) -> StatusResponse:
+        response = StatusResponse(False)
+
+        try:
+            if self.connect():
+                response.success = True
+            else:
+                response.error_message = "API key is missing"
+        except Exception as e:
+            response.error_message = str(e)
+
+        return response
+
+    def call_duckduckgo_api(self, params: Dict = None) -> Any:

```

Comment:
This is better to convert to API Table. Please check the docs for building the APIHandlers

Reply:
https://docs.mindsdb.com/contribute/app-handlers

Reply:
the response are in table  
![image](https://github.com/mindsdb/mindsdb/assets/119131588/1937cdd1-3cd8-4fa3-9f45-0e37452976fa)
![image](https://github.com/mindsdb/mindsdb/assets/119131588/57e7e859-17fd-405d-9849-90cbef02be8c)
is this what you are talking about

Reply:
i just made the changes @ZoranPandovski please review it